### PR TITLE
Ports Guitar from UristMcStation 

### PR DESCRIPTION
**Pull request by tigercat2000**
UristMcStation/UristMcStation#191

Self-explanatory, adds a guitar, similar to the violin, but different sounds

Example video; https://youtu.be/TEqDhc5A-MM

Update; The guitar is now obtainable by two methods:
 * Tablecrafting, requires 5 wood planks, 6 cable coil, a screwdriver, and a pair of wirecutters
 * Cargo store: Like the Violin, but slightly more expensive.

### CODE REVIEW

**Comment by Spacemanspark** [comment]
YES. YES.
:+1: :100: :100:

**Comment by Glloyd** [comment]
It doesn't look like you've added an icon. Just use the icon from us man, it matches the in_hands.

**Comment by tigercat2000** [comment]
@Glloyd I think our icon fit a bit better with the violin, heh

**Comment by Glloyd** [comment]
If it's the one in the video, don't go with it. It looks kinda bad IMO.

**Comment by Fox-McCloud** [comment]
I will only support this.....

if you add a custom hitsound akin to what's heard when the engie smashes his guitar

=p

**Comment by tigercat2000** [comment]
@Fox-McCloud if you know how to do hitsounds and have a sound file, gladly

**Comment by Glloyd** [comment]
Fair enough, but the violin is ugly too. Personally, I'd rather have something that looks good individually than something that matches, but that's just me.

**Comment by M3hillus** [comment]
The question remains. Acoustic, or ELECTRIC BOOGALOO!

**Comment by Fox-McCloud** [comment]
Acoustic forevaaaah

File: code/game/objects/items/devices/guitar.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	icon_override = 'icons/mob/in-hand/tools.dmi'
+	force = 10
+	var/datum/song/handheld/song
+
+/obj/item/device/guitar/New()
+	song = new("guitar", src)
+	song.instrumentExt = "ogg"
+
+/obj/item/device/guitar/Destroy()
+	del(song)
```
Comment (Iamgoofball) [inline code/game/objects/items/devices/guitar.dm]: >del(song)

don't you mean qdel(song)

Comment (tigercat2000) [inline code/game/objects/items/devices/guitar.dm]: can qdel even handle datums?

Comment (Fox-McCloud) [inline code/game/objects/items/devices/guitar.dm]: it can't, sadly--TG's garbage collector can; ours just throws a "HEY ONLY ATOM MOVEABLE!"

**Pull request merged**